### ISSUE TITLE
Poor contrast with some block colours: 

### ISSUE THREAD

**Issue by richard-delorenzi** [LLK/scratch-gui#6308]
### Expected Behavior
All text should be readable, and comply to [WCAG](https://www.w3.org/WAI/standards-guidelines/wcag/) accessibility standards.

### Actual Behavior

The text on some colour blocks is hard to read: Some of my students and I struggle to read some blocks. The worse is the custom `My Blocks`. 

Blocks with light colour fail to meet accessibility standards.

### Steps to Reproduce
Try to read the text on the blocks. It is worse on a projector, with high ambient light.

Use a WCAG tester, to check contrast.
 - [`Sounds`](https://webaim.org/resources/contrastchecker/?fcolor=FFFFFF&bcolor=CF63CF) Contrast ratio 3.31:1
 - [`Events`](https://webaim.org/resources/contrastchecker/?fcolor=FFFFFF&bcolor=FFBF00) Contrast ratio 1.65:1
 - [`Control`](https://webaim.org/resources/contrastchecker/?fcolor=FFFFFF&bcolor=FFAB19) Contrast ratio 1.89:1
 - [`My blocks`](https://webaim.org/resources/contrastchecker/?fcolor=FFFFFF&bcolor=FF6680) Contrast ratio 2.81:1
 - [`Variables`](https://webaim.org/resources/contrastchecker/?fcolor=FFFFFF&bcolor=FF8C1A) Contrast ratio 2.32:1

With a small change (black text for these blocks), we get it to pass, and be easy to read. Sounds also needs a subtle change to the background colour.

 - [`Sounds`](https://webaim.org/resources/contrastchecker/?fcolor=000000&bcolor=D77ED7) Contrast ratio 7.85:1
 - [`Events`](https://webaim.org/resources/contrastchecker/?fcolor=000000&bcolor=FFBF00) Contrast ratio 12.7:1
 - [`Control`](https://webaim.org/resources/contrastchecker/?fcolor=000000&bcolor=FFAB19) Contrast ratio 11.09:1
 - [`My blocks`](https://webaim.org/resources/contrastchecker/?fcolor=000000&bcolor=FF6680) Contrast ratio 7.46:1
 - [`Variables`](https://webaim.org/resources/contrastchecker/?fcolor=000000&bcolor=FF8C1A) Contrast ratio 9.01:1

### Operating System and Browser
All
